opendebates/views.py

```diff
@@ -186,6 +186,20 @@ def vote(request, id):
         voter.source = request.COOKIES['opendebates.source']
         voter.save()
 
+    if voter.user and voter.user != request.user:
```

Comment:
``request.user`` should never be ``None`` (it'll either be a User or an AnonymousUser), so I think we can skip testing that ``voter.user`` isn't None.  If it is, the next test will fail anyway.  Of course, we could keep the initial test in just to make this clearer; up to you.

I was wondering about comparing two Model objects directly for equality, but it turns out Django does what we want in this case - https://docs.djangoproject.com/en/dev/topics/db/queries/#comparing-objects.  So that's okay and I've learned something :-)